### PR TITLE
Breaking: Store floatbox instance on `floatbox-layer`

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@christopheschwyzer please review

File: client-vendor/after-body/jquery.floatbox/jquery.floatbox.js [hunk 1]
```diff
@@ -148,27 +148,28 @@
     }
   });
 
-  $.fn.floatOut = function(options) {
+  /**
+   * @param {String} methodOrOptions
+   * @param {...*} args
```
Comment (christopheschwyzer) [inline client-vendor/after-body/jquery.floatbox/jquery.floatbox.js]: What does this mean `{...*}`?

Comment (vogdb) [inline client-vendor/after-body/jquery.floatbox/jquery.floatbox.js]: Docu for infinite number of arguments. Please check http://usejsdoc.org/tags-param.html.

Comment (christopheschwyzer) [inline client-vendor/after-body/jquery.floatbox/jquery.floatbox.js]: Got it. Thanks for clarifying.

**Pull request merged**